packages/babel-plugin-syntax-import-attributes/src/index.ts

```diff
@@ -5,7 +5,7 @@ export interface Options {
 }
 
 export default declare((api, { deprecatedAssertSyntax }: Options) => {
-  api.assertVersion(REQUIRED_VERSION("^7.22.0"));
+  api.assertVersion(REQUIRED_VERSION("^7.22.0 || ^8.0.0-0"));
```

Comment:
I think this plugin should be kept Babel 7 only. We should remove the `deprecatedImportAssert` parser plugin in Babel 8 and then archive this syntax plugin.